packages/web-components/fast-element/tsconfig.json

```diff
@@ -1,16 +1,20 @@
 {
     "extends": "../../../tsconfig.json",
     "compilerOptions": {
-        "outDir": "dist",
-        "baseUrl": ".",
+        "declarationDir": "dist/dts",
+        "outDir": "dist/esm",
         "strictNullChecks": true,
-        "target": "es2015"
+        "target": "es2015",
+        "module": "ESNext",
+        "removeComments": true,
+        "types": [
+            "mocha",
+            "webpack-env"
```

Comment:
I don't see `@types/webpack-env` enumerated as a dependency - where is that coming from?

Reply:
Oops. That shouldn't be needed. I'll remove it.

Reply:
Double oops. It's actually needed in the browser-setup code. I'll add the dependency. Not sure why that wasn't there.